.travis.yml

```diff
@@ -6,40 +6,41 @@ notifications:
   email:
     on_success: never
 stages:
-- tests
-- e2e-helm-deployment-test
-- e2e-cloud-function-deployment-test
-- github-deploy
-- docker-deploy
-- gcp-install-release
+  - tests
+  - build
+  - e2e-helm-deployment-test
+  - e2e-cloud-function-deployment-test
+  - github-deploy
+  - docker-deploy
+  - gcp-install-release
 env:
   global:
   - TAG=$TRAVIS_TAG
   - VERSION_TAG=/^release-.*
   - DOCKER_CLI_EXPERIMENTAL=enabled
+  - GCP_PROJECT_ID="dynatrace-gcp-extension"
+  - GCR_NAME="gcr.io/${GCP_PROJECT_ID}/dynatrace-gcp-function"
 jobs:
   include:
   - stage: tests
     name: Tests
     language: python
-
+    python:
+      - "3.8"
+    env:
+      - PYTHONPATH="$TRAVIS_BUILD_DIR/src:$TRAVIS_BUILD_DIR/tests"
+    addons:
+      apt:
+        packages:
+          - default-jre
     cache:
```

Comment:
Non-review question... does `cache` allow to preserve installed packages from stage to stage?

Reply:
looks like true https://docs.travis-ci.com/user/caching/#caching-directories-bundler-dependencies